flow/templates/anvil.sh

```diff
@@ -0,0 +1,33 @@
+{# Templated in accordance with: https://www.rcac.purdue.edu/knowledge/anvil/ #}
+{% extends "slurm.sh" %}
+{% block tasks %}
+    {% if resources.ngpu_tasks %}
+        {% if not ('gpu' in partition or force) %}
+            {% raise "GPU operations require a gpu partition!" %}
+        {% endif %}
+        {% if resources.ngpu_tasks > 4 %}
```

Comment:
Is this true, they don't list a GPU node limit for `gpu` in https://www.rcac.purdue.edu/knowledge/anvil/run/partitions.

Reply:
Yes, this is true:
```
salloc -Admr140129 --nodes=2 --ntasks-per-node=4 --gpus-per-node=4  --partition shared -t 1:00:00
salloc: error: QOSMaxNodePerJobLimit
```
Would you prefer that we remove this error from flow and allow `sbatch` to notify the user?

Reply:
Although, they do provide a multi-node example script: https://www.rcac.purdue.edu/knowledge/anvil/run/examples/slurm/gpu#how_to_use_slurm_to_submit_a_multi-node_gpu_program